deploy/helm/kubecf/assets/operations/instance_groups/acceptance-tests.yaml

```
@@ -40,4 +40,12 @@
       serial: true
     vm_type: minimal
 
+
+{{- if .Values.testing.cf_acceptance_tests.delete_pod }}
+- type: replace
+  path: /instance_groups/name=acceptance-tests/env?/bosh/agent/settings/labels/delete?
```

```suggestion
  path: /instance_groups/name=acceptance-tests/env?/bosh/agent/settings/labels/delete
```
The `?` applies to the key and any subsequent keys; no need to repeat it.